app/javascript/components/Login.jsx

```diff
@@ -20,14 +22,23 @@ class Login extends Component{
         });
     }
 
-    onSubmitForm(e){
-        
+    handleSubmit(e){
+        e.preventDefault();
+
+        this.onSubmitForm(e.target);
+    }
+
+    async onSubmitForm(e){
+        await loginUser({
+            username: this.state.username, 
+            password_digest: this.state.password_digest
+        }, this.props.history)
```

Comment:
We shouldn't be passing `this.props.history` here. I know this is here because we're redirecting the user later, but there are other ways to handle redirecting. Let's remove it for now and handle the redirecting in a different PR.